manifests/service.pp

```diff
@@ -40,7 +40,7 @@
   if $service_manage {
     case $::osfamily {
       'OpenBSD': {
-        service { 'nginx':
+        service { 'puppet_nginx':
```

Comment:
I wonder if we should just use ```$service_name``` and drop the ```name => $service_name```.

Reply:
Same thing as name defaults to the resource name, just more explicit.  Would be tidier that way, I agree.

Reply:
IMHO the biggest benefit is that there's no risk of overlapping resource names for different services.

Reply:
Agreed, updated with https://github.com/voxpupuli/puppet-nginx/pull/1159/commits/7e7dbf66deeae822ec3d99174516684b15d7be32